### ISSUE TITLE
Update `Our Journey` Image to include Series C Annoucement

### ISSUE THREAD

**Issue by phillipedwards** [pulumi/pulumi-hugo#3569]
### What happened?

Series C announcement is not included in our `History` section of pulumi.com/about/#history. Specifically, the `Our journey` image is out of date.

### Example

![image](https://github.com/pulumi/pulumi-hugo/assets/25461821/ed66e419-e689-4589-a38f-cef429809305)

### Output of `pulumi about`

n/a

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by desteves** [comment]
@pulumi/developer-advocates Thoughts on moving this issue to marketing?

**Comment by cnunciato** [comment]
@desteves sounds great, and cc @anita-trimbur who may be the keeper of this SVG (or know who is -- last time we updated, @jonelleb helped us out with it).

**Comment by ericrudder** [comment]
good idea ... @anita-trimbur might as well add ESC and Plat Teams to timeline too!
